tests/check/check_signed_video_auth.c

```diff
@@ -2202,18 +2202,23 @@ START_TEST(golden_sei_principle)
 
   SignedVideoCodec codec = settings[_i].codec;
   SignedVideoReturnCode sv_rc;
-  signed_video_t *sv = signed_video_create(codec);
-  ck_assert(sv);
   char *private_key = NULL;
   size_t private_key_size = 0;
   // Setup the key
   sv_rc = settings[_i].generate_key(NULL, &private_key, &private_key_size);
   ck_assert_int_eq(sv_rc, SV_OK);
+  // Generate golden SEI
```

Comment:
Add an empty line before this comment to make a clear distinction between the different parts of the test.